InfinityCosmos/ForMathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -0,0 +1,13 @@
+import Mathlib.AlgebraicTopology.SimplexCategory
+import Mathlib.CategoryTheory.Limits.Shapes.Terminal 
+
+open CategoryTheory Simplicial SimplexCategory Limits
+
+namespace SimplexCategory
+
```

Comment:
Add an extra line here and then put a comment above your definition describing what it is, e.g.,
```
/-- The object `[0]` is terminal in `SimplexCategory`.-/
```

The comments above definitions have a special format, I believe, so that lean will include this information when someone hovers over the definition (or something like that).